tensorrt_llm/executor/result.py

```diff
@@ -661,3 +698,28 @@ def _topk_logprobs(logits: torch.Tensor, top_k: int,
 
     return LogProbsResult(prompt=prompt_logprobs,
                           generation=generation_logprobs)
+
+
+def _process_req_perf_metrics(req_perf_metrics_dict: Optional[dict[str, float]],
+                              output_length: int) -> Optional[dict[str, float]]:
+    stat = {}
+    if not req_perf_metrics_dict:
+        return stat
+    ttft = req_perf_metrics_dict.get(RequestEventTiming.FIRST_TOKEN_TIME, 0) - \
```

Comment:
These calculations seem fine to me. I noticed we default to 0, is there are scenario where we don't expect these keys in the dictionary? If any of these come out to be zero won't we return erroneous results?